addons/sale_timesheet/tests/test_project.py

```diff
@@ -129,3 +129,53 @@ def test_compute_cost_in_employee_mappings(self):
         employee_mapping.cost = self.employee_user.timesheet_cost
         employee_mapping.employee_id = self.employee_company_B
         self.assertEqual(employee_mapping.cost, self.employee_company_B.timesheet_cost)
+
+    def test_analytic_account_balance(self):
+        """
+            1) Add new billable project
+            2) Add Employee/SOL mapping in the project
+            3) Add Task and Timesheet with the same user
+            4) Assert analytic_account_balance is calculated
+        """
+
+        sale_order = self.env['sale.order'].create({
+            'partner_id': self.partner_b.id,
+        })
+
+        sale_line = self.env['sale.order.line'].create({
+            'name': self.product_delivery_timesheet1.name,
+            'product_id': self.product_delivery_timesheet1.id,
+            'product_uom_qty': 1,
+            'product_uom': self.product_delivery_timesheet1.uom_id.id,
+            'price_unit': self.product_delivery_timesheet1.list_price,
+            'order_id': sale_order.id,
+        })
+
+        unit_cost = 15
+        unit_amount = 6
+        expected_analytic_account_balance = unit_cost * unit_amount * -1
```

Comment:
Since the timesheet cost of the employee used in the creation of a employee mapping for the project below is also equal to 15. We could rewrite like this:
```suggestion
        unit_amount = 6
        expected_analytic_account_balance = self.employee_user.timesheet_cost * unit_amount * -1
```

What do you think? 🙂

Reply:
okay no problem